### ISSUE TITLE
Incompatible with gnome-shell 3.8.4-1

### ISSUE THREAD

**Issue by orschiro** [siefkenj/gnome-shell-windowlist#30]
With the recent update to gnome-shell 3.8.4-1 windowlist no longer works.

Can anyone confirm that?

![1](http://i.imgur.com/I9JfALp.png)

**Comment by siefkenj** [comment]
Did my push last night fix the problem?

**Comment by orschiro** [comment]
No. I cloned the repo to `.local/share/gnome-shell/extensions` but it does not even show up in the list of installed extensions.

**Comment by fazakis** [comment]
I have the same problem is there any way to fix that?
